components/dashboard/CheckUsername.tsx

```diff
@@ -33,6 +33,7 @@ const CheckUsername = () => {
       setShowUsernameModal(false);
     } catch (error) {
       toast.error("Error setting username");
+      setInvalidUsername(true);
```

Comment:
Doing this to show an error message when we are not following the rules. Although I think we should differentiate between invalid username errors and username taken errors if possible